syntax = "proto3";
package com.panelsw.ca;
import "errorHandling.proto";
option optimize_for = LITE_RUNTIME;

message UnzipDetails{

	enum UnzipFlags {
		never = 0;
		always = 1;
		unmodified = 2;

		delete = 16;

		createRoot = 32;
		onRollback = 64; // Used by the scheduler
	}

    bytes zipFile = 1;
    bytes targetFolder = 2;
	UnzipFlags flags = 3;
	ErrorHandling errorHandling = 4;
}
